README.md

```diff
@@ -34,14 +34,10 @@ oc create -f ssh-secret.yml
 ```
 
 #### Registry Credentials
-The CI pipeline requires credentials to push and/or pull from all authenticated
-registries. This includes:
-
-* Red Hat terms-based registry (registry.redhat.io)
-* The registry specified by the `registry` param (including the internal Openshift registry)
-
-The user must create an auth secret containing the docker config with all
-credentials included. For example:
+The CI pipeline can optionally be configured to push images to a remote private
```

Comment:
How can user not specify the credentials?
After I removed the parameters `registry` and workspace `credentials`, I've got the error 
```
[build-bundle : push] + buildah --storage-driver=vfs push --tls-verify=true --digestfile /workspace/source/image-digest image-registry.openshift-image-registry.svc:5000/redhat-isv/kogito-operator:1.6.0-ok docker://image-registry.openshift-image-registry.svc:5000/redhat-isv/kogito-operator:1.6.0-ok
[build-bundle : push] Getting image source signatures
[build-bundle : push] unauthorized: authentication required
[build-bundle : push] level=error msg="exit status 125"
```

Reply:
Are you specifying the `image_stream` param as well? It seems to be poorly named but it's the namespace the image gets pushed. You can remove that param and it should default to the current pipeline namespace.

Reply:
Could you please add one sentence, explaining how the parameters are different for pipeline pointing to the private registry?

Reply:
I added an example invocation for the internal registry and renamed the param.

Reply:
Thank you, that's great